src/main/resources/data.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO users (USERNAME, GENDER, EMAIL, ROLE, PASSWORD, DATE_OF_BIRTH, ENABLED) VALUES
+('Thanos','MALE','simpleMail637@gmail.com','USER', '$2y$04$tA374Kz2SkPzDM20xNESsO0yho0lSFJjeVjSIIXD9q4gOBHudr7Km','2000-01-01', TRUE);
+
+--пароль 111
```

Comment:
Why do you need it here?

Reply:
this is necessary if I suddenly forget the user password